app/models/order.py

```diff
@@ -7,7 +7,7 @@
 class OrderTicket(db.Model):
     __tablename__ = 'orders_tickets'
     order_id = db.Column(db.Integer, db.ForeignKey('orders.id', ondelete='CASCADE'), primary_key=True)
-    ticket_id = db.Column(db.Integer, db.ForeignKey('ticket.id', ondelete='CASCADE'), primary_key=True)
+    ticket_id = db.Column(db.Integer, db.ForeignKey('ticket.id', ondelete='CASCADE'), nullable=True)
```

Comment:
Then please check why `CASCADE` is failing to delete the rows and fix that. A `CASCADE` is supposed delete the row when the related parent row is deleted.